stdlib/REPL/test/repl.jl

```diff
@@ -150,1440 +98,21 @@ fake_repl(options = REPL.Options(confirm_exit=false,hascolor=true)) do stdin_wri
             readuntil(stdout_read, tmpdir_pwd)
             readuntil(stdout_read, "\n")
             readuntil(stdout_read, "\n")
```

Comment:
A little part of me wonders if the path on aarch64 is just a little bit too long, and causes tmpdir_pwd to be printed several times here. e.g. the same as #44972. The order of these reads is a bit nonsensical. Thus the actual `cd` might not have happened yet, until the added `@show` cause a context switch.

(you can prove this theory by checking when `get(ENV, "OLDPWD", "")` gets set here)

Reply:
Yeah, it's as you expected. A context switch is what triggers it to change. Not sure what the fix for this is. The paths don't look too too long